action.yml

```diff
@@ -84,6 +84,11 @@ inputs:
     required: false
     default: ''
 
+  compress_assets:
```

Comment:
I think provide several options for this would be better:
- `auto`: current behavior
- `off`: disable
- `zip`: all use zip for the compression, including Linux
Maybe other compression methods in the future.     

How's your option?